packages/account/CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 3.1.2 (2023-12-28)
+
+
+### Bug Fixes
+
+* use undefined in place of ! + check on limits returned by paymaster and throw ([0376901](https://github.com/bcnmy/biconomy-client-sdk/commit/0376901b7aec8c268a6a3c654d147335974d78f3))
+
```

Comment:
We're missing some:

https://github.com/bcnmy/biconomy-client-sdk/pull/342
https://github.com/bcnmy/biconomy-client-sdk/pull/340
https://github.com/bcnmy/biconomy-client-sdk/pull/337
https://github.com/bcnmy/biconomy-client-sdk/pull/336
https://github.com/bcnmy/biconomy-client-sdk/pull/335
https://github.com/bcnmy/biconomy-client-sdk/pull/332

Reply:
340 is not going into this release r7

Reply:
335 is already covered in this

Reply:
updated for the rest

Reply:
https://github.com/bcnmy/biconomy-client-sdk/commit/1868d71158f79c84cf97d09fdbbf22b455eecb7c